mlflow/protos/service.proto

```diff
@@ -471,6 +471,24 @@ service MlflowService {
       rpc_doc_title: "Log Batch",
     };
   }
+
+  // .. note::
+  //     Experimental: This API may change or be removed in a future release without warning.
+  rpc logModel (LogModel) returns (LogModel.Response) {
+    option (rpc) = {
```

Comment:
Why do we have both the preview & non-preview paths here? Happy to discuss offline

Reply:
Just because we have it for all the other endpoints. I did not want this to behave differently from the others (e.g. I am not sure if R client does not use preview still?) We should remove them all at once I think.